import-export-cli/cmd/importAPI.go

```diff
@@ -993,3 +1003,141 @@ func init() {
 	_ = ImportAPICmd.MarkFlagRequired("environment")
 	_ = ImportAPICmd.MarkFlagRequired("file")
 }
+
+// setupEndpoints will set up the endpoints accordingly, for the applicable type
+// @param environmentParams : Environment parameters from api_params.yaml
+// @return configData as a byte array
+// @return error
+func setupEndpoints(environmentParams *params.Environment) ([]byte, error) {
+	var configData []byte
+	var err error
+
+	// if the endpoint routing policy or the endpoints field is not specified
+	if environmentParams.EndpointRoutingPolicy == "" && environmentParams.Endpoints == nil {
+		// if endpoint type is Dynamic
+		if environmentParams.EndpointType == utils.DynamicEndpointType {
+			configData = []byte(utils.DynamicEndpointConfig)
+		} else if environmentParams.EndpointType == utils.AwsLambdaEndpointType { // if endpoint type is AWS Lambda
+			if environmentParams.AWSLambdaEndpoints == nil {
+				return nil, errors.New("Please specify awsLambdaEndpoints field for " + environmentParams.Name + " and continue...")
+			}
+			if environmentParams.AWSLambdaEndpoints.AccessMethod == utils.AwsLambdaRoleSuppliedAccessMethod {
+				environmentParams.AWSLambdaEndpoints.AccessMethod = utils.AwsLambdaRoleSuppliedAccessMethodForJSON
+			}
+			environmentParams.AWSLambdaEndpoints.EndpointType = utils.AwsLambdaEndpointTypeForJSON
+			configData, err = json.Marshal(environmentParams.AWSLambdaEndpoints)
+		} else {
+			return nil, errors.New("Please specify the endpoint routing policy or the endpoints field for " + environmentParams.Name + " and continue...")
```

Comment:
Does this exceed the maximum line length we maintain? If so let's break it into two lines. Long line lengths hinder the readability of code.

Reply:
@wasuradananjith Is this valid?

Reply:
For Go we have not enforced such maximum limit. However, let's split this into 2 lines to increase the readability.

Reply:
fixed via https://github.com/wso2/product-apim-tooling/pull/962/commits/7342c1b616a06cd866bdb948237b407b2f8fdb3c